packages/patternfly-4/react-charts/src/components/AreaChart/examples/CustomColorsChart.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { ChartArea, ChartGroup, ChartLegend, ChartVoronoiContainer } from '@patternfly/react-charts';
+import { ChartArea, ChartGroup, ChartLegend, ChartVoronoiContainer } from '../../index';
```

Comment:
If I remember correctly these are required for the docs package since files get copied over.